### ISSUE TITLE
Meta: add sponsor button in the repository

### ISSUE THREAD

**Issue by EzerArch** [micdoodle8/Galacticraft#3833]
Hey @radfast @micdoodle8, GitHub has a new (?) feature: sponsor button, so you link a Patreon page or something alike, as [Forge](https://github.com/MinecraftForge/MinecraftForge) did.

**Details:** https://help.github.com/en/github/building-a-strong-community/displaying-a-sponsor-button-in-your-repository

**Comment by radfast** [comment]
Yes, I saw this - that's a nice feature, and we would certainly welcome more support.  I feel it's up to @micdoodle8 to decide how we implement this though.
